backend/dataall/aws/handlers/sts.py

```diff
@@ -99,7 +99,7 @@ def get_external_id_secret(cls):
         """
         return SessionHelper.get_secret(
             secret_name=f'dataall-externalId-{os.getenv("envname", "local")}'
-        )
+        ) or 'localNoExternalId'
```

Comment:
I am personally not a big fan of mixing the production code with a code for local development.
If creating an own secret as a part of setting up a environment for playground is not desired I would think of making it configurable so the customer can decide whether it is needed to use external id additionally or not.

Then for the local setup we can set this parameter to false.

Reply:
I agree with the above.

Reply:
Okey makey. I was also not very convinced :)  Closing this now